code/src/index.css

```diff
@@ -1,13 +1,170 @@
+* {
+  box-sizing: border-box;
+  margin: 0px;
+}
+
 body {
-  margin: 0;
   font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen",
     "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue",
     sans-serif;
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
+  background-color: #111;
 }
 
 code {
   font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
     monospace;
 }
+
+header {
+  border-bottom: 1px solid #333;
+}
+
+.header-title {
+  font-family: 'Helvetica', sans-serif;
+  font-size: 20px;
+  font-weight: bold;
+  color: #FFF;
+  margin-left: 19vw;
+  margin-bottom: 0.5vw;
+  margin-top: 0.5vw;
+}
+
+/* STYLING APP.JS */
+.all-albums {
+  margin-left: 15vw;
+  margin-right: 15vw;
+}
+
+/* STYLING ALBUMCARD.JS */
+
+.album-container {
+  width: 100%;
+}
+
+.wrapper-for-children {
+  width: 100%;
+  padding: 20px;
+}
+
+/* STYLING ALBUM COVER */
+.album-cover {
+  width: 100%;
+}
+
+.overlay {
+  position: absolute;
+  top: 0;
+  right: 0;
+  bottom: 0;
+  left: 0;
+  opacity: 0;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  transition: 0.4s ease;
+}
+
+.overlay:hover {
+  opacity: 100;
+  background: rgba(44, 44, 44, 0.7);
+}
+
+.overlay-relative {
+  position: relative;
+}
+
+/* STYLING ALBUM NAME AND ARTIST */
+.album-name {
+  font-family: 'Helvetica', sans-serif;
+  font-size: 14px;
+  font-weight: bold;
+  color: #FFF;
+  text-decoration: none;
+}
+
+.album-name:hover {
+  border-bottom: 1px solid #FFF;
+}
+
+.artist-name {
+  font-family: 'Helvetica', sans-serif;
+  font-size: 14px;
+  font-weight: normal;
+  color: #A0A0A0;
+  text-decoration: none;
+}
+
+.artist-name:not(:last-child)::after {
+  content: ", ";
+  color: #a0a0a0;
+}
+
+.artist-name:nth-last-child(2)::after {
+  content: " & ";
+  color: #a0a0a0;
+}
+
+.artist-name:hover {
+  color: #FFF;
+  border-bottom: 1px solid #FFF;
+}
+/* STYLING ICONS */
+.icons-container {
+  display: flex;
+  justify-content: center;
+  column-gap: 20px;
```

Comment:
Cool, I didn't know that it's possible to use column-gap with flex-box! I'll definitely keep this in mind since gap is one of my favorites with grid